source/users/sessions.txt

```diff
@@ -310,13 +310,10 @@ Configure Refresh Token Expiration
 
 By default, refresh tokens expire 60 days after they are issued. You can
 configure this time for your App's refresh tokens to be anywhere between
-30 minutes and 180 days. There are two exceptions:
+30 minutes and 180 days.
 
-- Anonymous user refresh tokens do not expire, though anonymous user
-  accounts are deleted after 90 days of inactivity.
-
-- Custom JWT refresh token expiration is determined by the ``exp`` value
-  of the user's JWT.
+Anonymous user refresh tokens do not expire. However, Anonymous user
```

Comment:
Not blocking for this review, but I wonder if this is still true, or if a configured refresh token expiration (or even the default refresh token expiration) overrides this?

Reply:
Checked with eng - still true but we had a slight error here anyway that I fixed